100.en.adoc

```diff
@@ -2,116 +2,118 @@
 
-Environmental DNA is thus a sample type, not a method. The starting point for eDNA investigations, therefore, includes DNA taken from any environmental sample where the DNA of a captured individual is not specifically targeted in the field. This includes water, soil, sediment and air, but also stool samples and tissue (plant/animal) where the host DNA is not targeted (https://doi.org/10.1093/oso/9780198767220.001.0001[Taberlet et al. 2018^]). Note, however, that choice of PCR primers sets taxonomic limits and introduces biases to signals of community compositions and abundances. A number of analytical methods exist for studying environmental DNA. You can divide these into two main types where one wants to 1) detect a specific organism or 2) describe a community of a range of organisms. Different methods of analysis will generate different types and volumes of data. Most often DNA concentrations are low, and technical and biological replicates should be included to validate species detection.
+eDNA is thus a sample type, not a method, including DNA derived from any environmental sample rather than from the capture and sequencing of a targeted individual. This includes water, soil, sediment and air, but also stool samples and tissue (plant/animal) where the host DNA is not targeted (https://doi.org/10.1093/oso/9780198767220.001.0001[Taberlet et al. 2018^]). Note, however, that choice of PCR primers sets taxonomic limits and introduces biases to signals of community compositions and abundances. A number of analytical methods exist for studying environmental DNA. These can be divided into two main classes: 1) those which aim to detect a specific organism and 2) those which describe a community of a range of organisms. Different methods of analysis will generate different types and volumes of data. Most often DNA concentrations are low, and technical and biological replicates should be included to validate species detection.
```

Comment:
Slightly dangling 'this' to start the second sentence. 'These samples…' or 'Such samples…'?